Example/Sources/ConversationViewController.swift

```diff
@@ -322,6 +318,11 @@ extension ConversationViewController: MessagesDisplayDelegate {
 //        let configurationClosure = { (view: MessageContainerView) in}
 //        return .custom(configurationClosure)
     }
+    
+    func configureAvatarView(avatarView: AvatarView, for message: MessageType, at indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView) {
```

Comment:
Nit but can we make this `configureAvatarView(_ avatarView: ...`. The `avatarView` is redundant